src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/GetProjectCommand.cs

```diff
@@ -50,24 +52,26 @@ protected override void ProcessRecordCore()
 
             if (All.IsPresent)
             {
-                var projects = EnvDTESolutionUtility.GetAllEnvDTEProjects(DTE);
+                VsSolutionManager.EnsureSolutionIsLoaded();
+                var projects = VsSolutionManager.GetAllVsProjectAdapters().Select(p => p.Project);
```

Comment:
I think we even don't need dte project for this command, it just write out project name, type, and full path which are easily accessible in deferred mode. 

You may not need to fix it right away, but may be, add a TODO to do it later.